src/ex3/ItemManager_cli.js

```diff
@@ -0,0 +1,115 @@
+import {
```

Comment:
why do you need this file and class extension ?
just use the ItemManager.js you already created in previous exercises

You have a lot of code duplicate between the files and you could have just refactored the old one to be able to handle different sources